### ISSUE TITLE
Accessibility issue: aria-atomic="true" missing in charts where aria-live="assertive" is used

### ISSUE THREAD

**Issue by ermcox** [highcharts/highcharts#19572]
#### Expected behaviour
SiteImprove is reporting that any use of aria-live="assertive" must also use aria-atomic="true". We use High Charts at Indiana University Alumni Association and Indiana University Foundation for annual reports and other things. Indiana University is requiring us to fix this item based on the Site Improve accessibility report.

We are using the accessibility module.

#### Actual behaviour
aria-atomic="true" missing

#### Live demo with steps to reproduce
The two charts on this page have aria-live="assertive", but are missing the aria-atomic="true" attribute.
https://alumni.iu.edu/about/annual-report.html

#### Product version

Highcharts JS v11.1.0 (2023-06-05)

#### Affected browser(s)
All

**Comment by kamil-musialowski** [comment]
Hi @ermcox , there is a pull request #19172  related to your issue that is already merged to master (but waiting for the next release to be available on CDN).

Could you please check, if the same issue occurs with code from master branch?
Use https://github.highcharts.com/ instead of https://code.highcharts.com/, to get the code from master branch.

**Comment by ermcox** [comment]
﻿Yes, that other request’s fix is the same and I can verify it fixes the error. Any idea when that will be pushed live? 


Eric Cox 
Senior Front-end Web Developer, Digital User Experience 
IU Advancement Marketing 
812-855-7221 
alumni.iu.edu <http://alumni.iu.edu/> / iufoundation.iu.edu <https://iufoundation.iu.edu/> 

This e-mail message, including all attachments, is private and may contain confidential and privileged information. If you are not the intended recipient, do not use, disseminate, disclose, or copy this information. Please notify the sender by reply e-mail and immediately delete this message.​ 


From: Kamil Musiałowski ***@***.***>
Date: Wednesday, August 16, 2023 at 1:18 PM
To: highcharts/highcharts ***@***.***>
Cc: Cox, Eric Michael ***@***.***>, Mention ***@***.***>
Subject: [External] Re: [highcharts/highcharts] Accessibility issue: aria-atomic="true" missing in charts where aria-live="assertive" is used (Issue #19572) 

You don't often get email from ***@***.*** Learn why this is important <https://aka.ms/LearnAboutSenderIdentification> 

This message was sent from a non-IU address. Please exercise caution when clicking links or opening attachments from external sources. 


Hi @ermcox <https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fermcox&amp;data=05%7C01%7Cermcox%40iu.edu%7Ca5207971d02743b685ae08db9e7cdbdd%7C1113be34aed14d00ab4bcdd02510be91%7C0%7C0%7C638278031337023251%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=5xx03MMjemhDAXAIqkr9LrCBafirk3MeiVtTXbGqNbo%3D&amp;reserved=0> , there is a pull request #19172 <https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fhighcharts%2Fhighcharts%2Fpull%2F19172&amp;data=05%7C01%7Cermcox%40iu.edu%7Ca5207971d02743b685ae08db9e7cdbdd%7C1113be34aed14d00ab4bcdd02510be91%7C0%7C0%7C638278031337023251%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=n8Grs1SUx12oARr%2FPu4TYocKwehCOpNznoR2KXwETdE%3D&amp;reserved=0> related to your issue that is already merged to master (but waiting for the next release to be available on CDN). 
Could you please check, if the same issue occurs with code from master branch?
Use https://github.highcharts.com/ <https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.highcharts.com%2F&amp;data=05%7C01%7Cermcox%40iu.edu%7Ca5207971d02743b685ae08db9e7cdbdd%7C1113be34aed14d00ab4bcdd02510be91%7C0%7C0%7C638278031337023251%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=K%2FRvPcpcCJROAQHuWummwtmJJYluAsl3zoKBa2rEpsk%3D&amp;reserved=0> instead of https://code.highcharts.com/ <https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.highcharts.com%2F&amp;data=05%7C01%7Cermcox%40iu.edu%7Ca5207971d02743b685ae08db9e7cdbdd%7C1113be34aed14d00ab4bcdd02510be91%7C0%7C0%7C638278031337023251%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=GuJxuBblRkNQ3PM9Uv2BLzru2Lqg7Hx3aTjDe5rGIoI%3D&amp;reserved=0>, to get the code from master branch.

**Comment by karolkolodziej** [comment]
I'm going to close this for now as it's been solved. 
Nevertheless, we will let you know as soon as we get information about the next release!

**Comment by kamil-musialowski** [comment]
@ermcox The next release with the PR included should be ready within the next 2 - 3 weeks. In the meantime, I have prepared a quick plugin for you to use in your existing projects.

Please try to use it on your side and let me know if everything is fine, or you need some further assistance.
https://jsfiddle.net/BlackLabel/av82o1sg/
